content/docs/releases/v1.1.0.md

```diff
@@ -0,0 +1,84 @@
++++
+title = "Release v1.1.0"
```

Comment:
Shouldn't the version be "v1.1.1" here?

Reply:
This is mostly to keep with the previous pattern.